docs/docs/security/cves.mdx

```diff
@@ -4,15 +4,30 @@ hide_title: true
 sidebar_position: 2
 ---
 
+#### Version 3.0.2, 2.1.3
+
+| CVE            | Title                                                       |                   Affected |
+|:---------------|:------------------------------------------------------------|---------------------------:|
```

Comment:
@dpgaspar We can assume that all fixes in 2.1.3 are present in 3.0.2. I suggest moving these to the 2.1.3 block and keep the incremental version order.

Reply:
could give the wrong impression that these fixes are included on 3.0.0 or 3.0.1

Reply:
You're right! Should we copy/duplicate them and keep the version order? It looks weird that 2.1.3 is defined twice.